spec/acceptance/init_task_spec.rb

```diff
@@ -1,7 +1,7 @@
 # run a test task
 require 'spec_helper_acceptance'
 
-describe 'apache tasks', if: puppet_version =~ %r{(5\.\d\.\d)} && fact('operatingsystem') != 'SLES' do
+describe 'apache tasks', if: puppet_version =~ %r{(5\.\d\.\d)} && host_inventory['facter']['os']['name'] != 'sles' do
```

Comment:
That puppet regex is rather specific. I suspect it'll break on `5.5.10` I think. Probably good to be safe and use `\d+`